src/index.html

```diff
@@ -6,8 +6,45 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div class="card" data-qa="card">
+
+      <a href="#" class="card__link link">
+        <img class="link__image" src="./images/imac.jpeg"
+        alt="Imac"
+        >
+      </a>
+
+      <a href="#" class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
+
+      <div class="card__code">Product code: 195434</div>
```

Comment:
Would be great to use semantic tags as much as possible
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/5739bdfe-e6a6-414a-acf9-2d70a62bb238)